model/atmosphere/diffusion/tests/diffusion_stencil_tests/test_apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence.py

```diff
@@ -13,7 +13,9 @@
     apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence,
 )
 from icon4py.model.common import dimension as dims
-from icon4py.model.common.test_utils.helpers import StencilTest, random_field, zero_field
+from icon4py.model.common.utils.data_allocation import random_field, zero_field
```

Comment:
why not importing `from icon4py.model.common.utils import data_allocation as data_alloc` like in other files?

Reply:
This answer applies to all comments related to the `import`s style:

I agree with the comments about wrong import styles but in this PR I only tried to adapt the imports moved to a newer location. Fixing the style of the imports which didn't follow the coding conventions is postponed to a later PR, to avoid making this PR even larger, and also to minimize merge conflicts with other open PRs.

Reply:
I disagree as this PR is large anyway and doing the imports correctly now would be more time-efficient IMO

Reply:
I want to try an automated approach to fix the style of some imports, so ideally we could be able to save some time...